### DISCUSSION TITLE
Coloring wires

### DISCUSSION THREAD

**Discussion by Navelion** [ra3xdh/qucs_s#591] [Q&A]
Hello, is there any way of coloring wires? Or would you implement these option in future? Beacose it is handy when you deal with a large  scheme. Especialy when you want to reasure that wires connected correct ways.

**Comment by zergud** [comment]
you described two different cases:
1. show unconnected wire in a different color
this can be implemented
2. set color to certain wire
I don't think it's really necessary

there was a request for customizable colors for all schematics primitives, such is pins, symbols, wires, text... according to dark/light theme

**Reply by Navelion** [reply]
What I meant is that when you have a lot of wires in a circuit, it's much easier to keep track of which is connected where when they are different colors. If it is hard to implement ok. But if it possible it is good QoL improvement.
